leetcode/generator_md.py

```diff
@@ -0,0 +1,48 @@
+class LeetCodeSol:
```

Comment:
Your Pull Request title: LEETCODE Added reverse-linked-list

Check result:

* Pull Request title must contain group number in ['1021', '1022', '1013', '2021', '2022']